docs/source/markdown/podman-network.unit.5.md.in

```diff
@@ -0,0 +1,197 @@
+% podman-network.unit(5)
+
+# NAME
+
+podman\-network.unit - systemd unit files for managing Podman networks using Quadlet
+
+# SYNOPSIS
+
+*name*.network
+
+# DESCRIPTION
+
+Network files are named with a `.network` extension and contain a section `[Network]` describing the
+named Podman network. The generated service is a one-time command that ensures that the network
+exists on the host, creating it if needed.
+
+By default, the Podman network has the same name as the unit, but with a `systemd-` prefix, i.e. for
+a network file named `$NAME.network`, the generated Podman network is called `systemd-$NAME`, and
+the generated service file is `$NAME-network.service`. The `NetworkName` option allows for
+overriding this default name with a user-provided one.
+
+Please note that stopping the corresponding service will not remove the podman network.
```

Comment:
I guess you copied it and it is incorrect in the current file. By default, the network is not deleted. But, users can set `NetworkDeleteOnStop=true`

Reply:
Not sure I follow. The current text says "will *not* remove the podman network.", which is aligned with what you said - `the network is not deleted`.